packages/context/src/objects/objects.ts

```diff
@@ -43,6 +43,40 @@ export type ObjectInstance = {
     [key: string]: unknown;
 };
 
+/**
+ * @typedef {object} FormioInput - creates a new type named 'FormioInput'
+ * This will be used if a form field needs to be configured for custom widget
+ * @property { FormioInput } components - a property of FormioInput type groups different sets of fields together into separate sections.
+ */
+export type FormioInput = {
+    label: string;
+    type: string;
+    key: string;
+    initialValue?: unknown;
+    property?: Property;
+    input?: boolean;
+    placeholder?: string;
+    description?: string;
+    tooltip?: string;
+    prefix?: string;
+    suffix?: string;
+    showCharCount?: boolean;
+    readOnly?: boolean;
+    isMultiLineText?: boolean;
+    data?: {
+        resource?: string;
```

Comment:
What is `resource`?

Reply:
This is being added by FormIO, removing it will not have any effect on the form.

Reply:
Removed.